package.json

```diff
@@ -15,6 +15,7 @@
     "generate:aws-control": "bun ./scripts/generate-aws-control.ts",
     "publish:npm": "bun run --filter alchemy publish:npm",
     "test": "bun ./alchemy/test/run.ts",
+    "test:vitest": "bun ./alchemy/test/run.ts --vitest",
```

Comment:
i think we can just replace test, we don't need a test:vitest because the test codes explicitly use vitest now